Gemfile

```diff
@@ -3,9 +3,8 @@ source 'https://rubygems.org'
 gem 'active_model_serializers'
 gem 'amazing_print'
 gem 'bootsnap'
-gem 'carrierwave'
+gem 'carrierwave', '>= 3.1.0.beta'
```

Comment:
this is just for the rails 7.2 compatibility. In reality we are going to remove carrierwave, which is the library for image uploading, and reimplement it with the official Rails lib now that it has it.